modules/nested-tabs/assets/js/base-nested-tabs.js

```diff
@@ -176,37 +155,28 @@ export default class BaseNestedTabs extends Base {
 		return this.elements.$tabTitles.filter( '[data-tab="' + tabIndex + '"]' ).hasClass( this.getSettings( 'classes.active' ) );
 	}
 
-	bindEvents() {
-		this.elements.$tabTitles.on( {
-			keydown: ( event ) => {
-				// Support for old markup that includes an `<a>` tag in the tab
-				if ( jQuery( event.target ).is( 'a' ) && `Enter` === event.key ) {
-					event.preventDefault();
-				}
+	addCollapseClassToItems( args ) {
+		if ( elementorFrontend.isEditMode() ) {
+			const $widget = this.$element,
+				$removed = this.findElement( '.e-collapse' ).remove();
 
-				// We listen to keydowon event for these keys in order to prevent undesired page scrolling
-				if ( [ 'End', 'Home', 'ArrowUp', 'ArrowDown' ].includes( event.key ) ) {
-					this.handleKeyboardNavigation( event );
-				}
-			},
-			keyup: ( event ) => {
-				switch ( event.code ) {
-					case 'ArrowLeft':
-					case 'ArrowRight':
-						this.handleKeyboardNavigation( event );
-						break;
-					case 'Enter':
-					case 'Space':
-						event.preventDefault();
-						this.changeActiveTab( event.currentTarget.getAttribute( 'data-tab' ), true );
-						break;
-				}
-			},
-			click: ( event ) => {
-				event.preventDefault();
-				this.changeActiveTab( event.currentTarget.getAttribute( 'data-tab' ), true );
-			},
-		} );
+			let index = 1;
+
+			this.findElement( '.e-con' ).each( function() {
+				const $current = jQuery( this ),
+					$desktopTabTitle = $widget.find( `.e-n-tabs-heading > *:nth-child(${ index })` ),
+					mobileTitleHTML = `<div class="e-n-tab-title e-collapse" data-tab="${ index }" role="tab">${ $desktopTabTitle.html() }</div>`;
```

Comment:
Generalize class names, pass as variables in an `args` object.
E.g.:
```
args = {
	headingClassName: 'e-n-tabs-heading',
	titleClassName: 'e-n-tab-title'
};
```

Reply:
Took the data from the `setting` object